cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -233,13 +233,14 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
         }
 
         /**
-         * Asynchronously reconciles the cluster CA secret.
-         * The secret has have the name determined by {@link AbstractModel#getClusterCaName(String)}.
-         * Within the secret the current certificate is stored under the key {@code cluster-ca.crt}
-         * and the current key is stored under the key {@code cluster-ca.crt}.
-         * Old certificates which are still within their validity are stored under keys named like {@code cluster-ca-<not-after-date>.crt}, where
+         * Asynchronously reconciles the cluster and clients CA secrets.
+         * The cluster CA secret has to have the name determined by {@link AbstractModel#getClusterCaName(String)}.
+         * The clients CA secret has to have the name determined by {@link KafkaCluster#getClientsCaName(String)}.
+         * Within both the secrets the current certificate is stored under the key {@code ca.crt}
+         * and the current key is stored under the key {@code ca.key}.
+         * Old certificates which are still within their validity are stored under keys named like {@code ca-<not-after-date>.crt}, where
          * {@code <not-after-date>} is the ISO 8601-formatted date of the certificates "notAfter" attribute.
-         * Likewise, the corresponding keys are stored under keys named like {@code cluster-ca-<not-after-date>.key}.
+         * Likewise, the corresponding keys are stored under keys named like {@code ca-<not-after-date>.key}.
          */
         Future<ReconciliationState>  reconcileClusterCa() {
```

Comment:
`reconcileClusterCa()` → `reconcileCas()` if we're going to reconcile them both here.